snippets/xml_snippets.json

```diff
@@ -415,10 +415,17 @@
         ],
         "description": "Add link tag to include stylesheet"
     },
+    "Self-closing field tag": {
+        "prefix": "oofield",
+        "body": [
+            "<field name=\"${1}\"${20}/>${50}"
```

Comment:
`<field name=\"${1}\"/>`

Reply:
It's more useful having a tab stop after the double quotes, to be able to add more attributes. Also a tab stop in the end allows you to just hit "enter" and continue in the next line (although I wouldn't mind to remove it since I'm used to use the shortcut to add a line at the end, but I was thinking in users that don't know it).

Reply:
Why not? `<field name=\"${1}\"${0}/>`

Reply:
Oh I forgot that $0 is the end always, sorry!

Reply:
Ah and about using $2 or $20, I prefer using $20 in my personal snippets because it leaves room for injecting new tab stops without altering the snippet body. No problem to use $2 if you prefer it.